examples/java/aws/imports/constraints.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
These were generated by git hooks. I'm not sure if that's intentional. I can remove them if not.